modules/jsoncons/1.0.0/presubmit.yml

```
@@ -0,0 +1,15 @@
+matrix:
+  platform:
+  - centos7
+  - debian10
+  - ubuntu2004
+  - macos
+  - windows
+  bazel: ["7.x", "8.x"]
+tasks:
+  verify_targets:
+    name: Verify build targets
+    bazel: ${{ bazel }}
+    platform: ${{ platform }}
+    build_targets:
+      - '@jsoncons//:jsoncons'
```

```suggestion
      - '@jsoncons//:jsoncons'
  build_flags:
    - --features=parse_headers
    - --process_headers_in_dependencies
```
provides CI coverage for header-only libraries.